src/Durable/DurableTimer.cs

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.Durable
+{
+    using System;
+    using System.Linq;
+    using System.Threading;
+
+    internal class DurableTimer
+    {
+        private readonly ManualResetEvent _waitForStop = new ManualResetEvent(initialState: false);
+
+        public void CreateTimerAndStop_OrContinue(OrchestrationContext context, DateTime fireAt)
```

Comment:
Let's remove the underscore from the method name to stay consistent with the de facto naming convention. We make an exception for unit test names only.

Reply:
Sounds good. My main concern is that the grouping of terms might be ambiguous, i.e. CreateTimer and (Stop or Continue) vs. (CreateTimerAndStop) or Continue. I am changing the name to StopAndCreateTimerOrContinue. Let me know if you see a similar issue in this name.